pipes/rules/metagenomics.rules

```diff
@@ -29,16 +30,29 @@ rule kraken:
     resources: cores=int(config.get("number_of_threads", 1))
     params: numThreads=str(config.get("number_of_threads", 1))
     run:
+        shell("pyenv which python")
+        shell("echo \n\n\n\n\n\n\n")
         shell("{config[bin_dir]}/metagenomics.py kraken {input} {config[kraken_db]} --outReads {output.reads} --outReport {output.report} --numThreads {params.numThreads}")
 
-rule krona_import_taxonomy:
-    input: join(config["data_dir"], config["subdirs"]["metagenomics"], "{sample}.kraken.reads.gz")
-    output: join(config["data_dir"], config["subdirs"]["metagenomics"], "{sample}.kraken.krona.html")
-    shell:
-        """
-        {config[bin_dir]}/metagenomics.py krona {input} {config[krona_db]} {output} --noRank
-        """
+method_reads_ext = {
+    'diamond': 'diamond.lca.gz',
+    'kraken': 'kraken.reads.gz'
+}
 
+method_krona_columns = {
+    'diamond':  (1, 2),
```

Comment:
I hate to say this, but I think I'd prefer the diamond LCA output to throw in a blank column in front of each row, so that the format can _exactly_ match the kraken text output format (at least as far as krona is concerned). That should only add one byte per read to the uncompressed text file, but should make the output files far more interchangeable.